dcrtimed/backend/filesystem/filesystem_test.go

```diff
@@ -176,6 +176,89 @@ func TestGetDigests(t *testing.T) {
 	for i, gr := range grs {
 		if i < count-1 && (!bytes.Equal(gr.Digest[:], hashes[i][:]) ||
 			gr.ErrorCode != foundGlobal) {
+			t.Fatalf("invalid digest got %x want %x ErrorCode "+
+				"got %v want %v", gr.Digest[:], hashes[i][:],
+				gr.ErrorCode, foundGlobal)
+		}
+		if i >= count && gr.ErrorCode != backend.ErrorNotFound {
+			t.Fatalf("invalid ErrorCode got %x want %x",
+				gr.ErrorCode, backend.ErrorNotFound)
+		}
+	}
+}
+
+func TestGetDigestsFoundInPrevious(t *testing.T) {
```

Comment:
Can you add a comment that explains the details of what this is testing.  It would not be obvious what FoundInPrevious means to a developer that is looking at this without any context.